pbf/templates/game.html

```diff
@@ -73,10 +73,12 @@ <h3>Host Actions:</h3>
     </select>
 	  <select name="color" onChange="this.style.backgroundColor = this.options[this.selectedIndex].style.backgroundColor">
 	    <option value="random">Random color</option>
-	    {% for color, code in game.available_colors %}
+	    {% for color, freq, code in game.color_freq %}
+	    {% if freq == 0 %}
```

Comment:
when adding a new player: no strong opinion between excluding in-use colours entirely (what is done here) vs showing them with a warning (what is done for the reassignment dropdown)

Reply:
I used to not want to allow it because it's confusing, but with anyone being able to reassign I trust players to self-regulate.

Reply:
If it's to be allowed, it can just go in a separate pull request. This pull request can stay as-is. The reason is that this pull request purely adds new functionality that wasn't there before, without changing any existing functionality. What I'm pondering in this comment is change to existing functionality, which can be considered independently of the new functionality.